### ISSUE TITLE
Nanopolish calling methyaltion

### ISSUE THREAD

**Issue by xyy-xiao** [jts/nanopolish#1145]
Hi, 
I got pod5 files and fastq file, and want to call methylation. I know Nanopolish does not support pod5 files, so I used blue-crab to convert pod5 files to blow5 files. Then I used slow5tools to convert the blow5 files to fast5 files. But I got errors: 
[readdb] num reads: 5561098, num reads with path to fast5: 392000
fast5 files could not be located for 5169098 reads
[post-run summary] total reads: 375421, unparseable: 0, qc fail: 0, could not calibrate: 0, no alignment: 0, bad fast5: 5309727
Is there something wrong with my convert files?

**Comment by hasindu2008** [comment]
@xyy-xiao 

This is R10 or RNA004 data I believe? Nanopolish only supports R9 and that could be why. You may try [f5c](https://github.com/hasindu2008/f5c/). The command line and output will be the same as nanopolish.

**Comment by Tang-pro** [comment]
@hasindu2008 
hi, 
so Can f5c also calculate polyA length?

**Comment by hasindu2008** [comment]
Unfortunately, f5c only supports methylation calling and eventalign, not polyA length.

**Comment by Tang-pro** [comment]
@hasindu2008 
Okay ,thank you!
If it is RNA004, what methods can be used to calculate the polyA length?

**Comment by hasindu2008** [comment]
I think newer versions of ONT's Dorado have a polyA length programme built-in - note I have not tried this.

**Comment by Tang-pro** [comment]
@hasindu2008 

Yes, But this step is coducted by sequecing company, because I don't have GPU.
Okay, thank you again!
